src/tcgwars/logic/impl/gen7/LostThunder.groovy

```diff
@@ -1513,7 +1513,6 @@ public enum LostThunder implements LogicCardInfo {
               if(tar){
                 tar.each {pcs->
                   my.deck.search("Select a Pokémon Tool to attach to $pcs",cardTypeFilter(POKEMON_TOOL)).each{
-                    deck.remove(it)
```

Comment:
?

Reply:
I've made changes to the main-engine side of these changes that may be relevant. Basically, I am/want to use MoveCard instead of manually adding to/removing from card lists, because I've not seen that be anything but trouble while trying to implement certain cards.